save-core/src/commonMain/kotlin/org/cqfn/save/core/utils/TomlUtils.kt

```diff
@@ -76,5 +76,6 @@ private inline fun <reified T : PluginConfig> Path.createPluginConfig(
 fun createPluginConfigListFromToml(testConfigPath: Path): List<PluginConfig> =
         TomlParser(KtomlConf())
             .readAndParseFile(testConfigPath.toString())
-            .getRealTomlTables()
+            // Current function returns all tables (including subtables), however we need to extract only top level sections
+            .getRealTomlTables().filter { it.level == 0 }
```

Comment:
Should this check be moved inside `getRealTomlTables`? Or into a separate method `getTopLevelTomlTables`?

Reply:
This is `ktoml` function, so I filtered it in such way

Now I created function extension `getTopLevelTomlTables`